test-loop-tests/src/examples/resharding.rs

```diff
@@ -23,7 +23,7 @@ fn resharding_example_test() {
     let chunk_producer = "cp0";
     let validators_spec = ValidatorsSpec::desired_roles(&[chunk_producer], &[]);
     let genesis = TestLoopBuilder::new_genesis_builder()
-        .protocol_version(PROTOCOL_VERSION - 1)
+        .protocol_version(PROTOCOL_VERSION)
```

Comment:
Undo all changes of `PROTOCOL_VERSION - 1` to `PROTOCOL_VERSION` in files test-loop-tests/src/examples/resharding.rs, test-loop-tests/src/tests/global_contracts_distribution.rs, test-loop-tests/src/tests/resharding_v3.rs